src/azure-cli/azure/cli/command_modules/acr/repository.py

```diff
@@ -284,6 +285,10 @@ def acr_repository_update(cmd,
         json_payload.update({
             'writeEnabled': write_enabled
         })
+    if teleport_enabled is not None:
```

Comment:
Does the parameter only apply on repository? If the image parameter is `hello-world:latest`, what is the behavior?